cms/server/admin/templates/contest.html

```diff
@@ -81,6 +81,15 @@ <h1>Contest configuration</h1>
       </td>
       <td><input type="text" name="score_precision" value="{{ contest.score_precision }}"></td>
     </tr>
+    <tr>
+      <td>
+        <span class="info" title="Allow contestant to make unofficial submission after the participation window ended"></span>
+        Allow unofficial submission after participation ended
```

Comment:
Sorry, can we also update the text and the helptext?

Reply:
Done!